config/_shared.json

```
@@ -206,5 +206,10 @@
 	"theme_color": "",
 	"theme_color_admin_color_scheme_override": true,
 	"100_year_plan_calendly_id": "wpcom-100-years/30min",
-	"bilmur_url": "https://s0.wp.com/wp-content/js/bilmur.min.js"
+	"bilmur_url": "https://s0.wp.com/wp-content/js/bilmur.min.js",
+	"site_spec": {
+		"script_url": "http://widgets.wp.com/site-spec/bundle-1.0.0.umd.js",
+		"css_url": "http://widgets.wp.com/site-spec/style-1.0.0.css",
```

Let's point to the latest version:

```suggestion
		"script_url": "http://widgets.wp.com/site-spec/bundle-1.0.1.umd.js",
		"css_url": "http://widgets.wp.com/site-spec/style-1.0.1.css",
```

---

[d6f7b36](https://github.com/Automattic/wp-calypso/pull/105543/commits/d6f7b36164c5b6ab27f2f376925721973ec0d23a)